xml.c

```diff
@@ -252,7 +252,7 @@ xml_parse_share_user(xmlDoc *doc, xmlNode *item, struct share_user *user)
 			continue;
 		if (xml_parse_bool(doc, child, "group", &user->is_group))
 			continue;
-		if (xml_parse_bool(doc, child, "outsideenterpise", &user->outside_enterprise))
+		if (xml_parse_bool(doc, child, "outsideenterprise", &user->outside_enterprise))
```

Comment:
this would be an api-like change. if the misspelling is actually intentional, it should be flagged.

Reply:
This and other api-like changes are fine - this was a bug here.